documentation/docs/12-page-options.md

```diff
@@ -32,22 +32,29 @@ export const hydrate = false;
 
 ### prerender
 
-It's likely that at least some pages of your app can be represented as a simple HTML file generated at build time. These pages can be [_prerendered_](/docs/appendix#prerendering).
+It's likely that at least some routes of your app can be represented as a simple HTML file generated at build time. These routes can be [_prerendered_](/docs/appendix#prerendering).
 
-Prerendering happens automatically for any page with the `prerender` annotation:
+Prerendering happens automatically for any `+page` or `+server` file with the `prerender` annotation:
```

Comment:
should we add `JS` here to clarify that you can't add it to `+page.svelte`?
```suggestion
Prerendering happens automatically for any `+page` or `+server` JS file with the `prerender` annotation:
```

Reply:
I was hoping the code sample would make it clear — adding JS muddies the waters a bit in the case where people are using TypeScript

Reply:
There are some other places where it's `+page/layout` with the `.js` ending. We should probably make that consistent, either adding the `.js` file always or never (except for when we introduce it).